portal/src/main/scala/cromwell/pipeline/CromwellPipelineApp.scala

```diff
@@ -35,7 +35,6 @@ object CromwellPipelineApp extends App {
     import components.datastorageModule._
 
     pipelineDatabaseEngine.updateSchema()
```

Comment:
We should not ignore result of this `update` anymore. If it returns `Failure` then we should not start an application.

Reply:
@myazinn It seems to me that the exception should be caught on line 47 and terminate the application. Or else we need to implement it somehow differently.

Reply:
There is no exception that will be thrown. Try to add invalid migration and see what happens.
Probably you need to read something about the difference between `try {} catch {}` and `Try {}` in Scala.